presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/util/JsonUtils.java

```diff
@@ -42,7 +43,8 @@ private JsonUtils() {}
         try {
             byte[] json = Files.readAllBytes(path);
             ObjectMapper mapper = new ObjectMapperProvider().get()
-                    .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
+                    .enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+                    .enable(MapperFeature.ACCEPT_CASE_INSENSITIVE_ENUMS);
```

Comment:
Please extract this to separate commit and add a test for that.

Reply:
done